system_tests/lewis_emulators/ambrell_easy_heat/device.py

```diff
@@ -0,0 +1,51 @@
+from collections import OrderedDict
+
+from lewis.devices import StateMachineDevice
+
+from .states import DefaultState
+
+
+class SimulatedAmbrellEasyHeat(StateMachineDevice):
+    def _initialize_data(self) -> None:
+        """
+        Initialize all of the device's attributes.
+        """
+        self.connected = True
+
+        self.address = 3
+
+        self.heat_status = 1
+
+        self.id = 12345
+
+        self.data = (
+            str(self.address) + "," + str(self.heat_status) + "," + "285.0,280.0,1006,286,0,972"
+        )
```

Comment:
not anything wrong, just a stylistic choice/tip that f-string might make your life easier here
```suggestion
        self.data = f"{self.address},{self.heat_status},285.0,280.0,1006,286,0,972"
```

Reply:
Forgot to use f-string here.  Have since updated the formation of the data and status strings using one.